src/modules/common/apiCalls.js

```diff
@@ -68,7 +68,8 @@ export const getUserById = (id) => {
 };
 
 export const getParentById = (id) => {
-  return fetch(`https://kiddo-backend.herokuapp.com/api/v1/parents/${id}`);
+  return fetch(`https://kiddo-backend.herokuapp.com/api/v1/parents/${id}`
+  ).then((res) => res.json());
```

Comment:
😎

Reply:
DOHHH!!! So sorry you had to deal with that headache